axum-extra/CHANGELOG.md

```diff
@@ -7,13 +7,11 @@ and this project adheres to [Semantic Versioning].
 
 # Unreleased
 
-- **breaking:** `axum::extract::ws::Message` now uses `Bytes` in place of `Vec<u8>`,
```

Comment:
These changelogs disappeared by accident or did they move somewhere?

Reply:
Moved to `axum` changelog since that's where they belong (first commit).